### ISSUE TITLE
Allow setting custom image for the NATS server

### ISSUE THREAD

**Issue by wallyqs** [nats-io/nats-operator#90]
Should allow making it possible to set a custom image for the server, for example:

```
apiVersion: "nats.io/v1alpha2"
kind: "NatsCluster"
metadata:
  name: "example-nats-1"
spec:
  size: 3
  # version "1.3.0"
  image: "nats:1.3.0"
```

**Comment by onesolpark** [comment]
I think it would be great to add support for custom images..

I did made minimal changes to the codes to make it work in our environment 
(offline environment with no internet-access - Have to use a custom image stored in our private repo)

Will make a PR with the changes I made.

**Comment by wallyqs** [comment]
Fixed via #93
